src/Init/Meta.lean

```diff
@@ -115,6 +115,8 @@ def getRoot : Name → Name
 
 @[export lean_is_inaccessible_user_name]
 def isInaccessibleUserName : Name → Bool
+  -- note: this uses `✝` (`\u{271D}`) whereas everywhere else in the code
+  -- we use the text-variant selector `✝︎` (`\u{271D}\u{FE0E}`).
   | Name.str _ s   => s.contains '✝' || s == "_inaccessible"
```

Comment:
I now added this comment. I am partially concerned that these unicode variant selectors will be forgotten and will confuse a future developer, but it's up to the maintainers how you want to document this correctly...